crates/bindings-csharp/Codegen/Module.cs

```diff
@@ -129,17 +160,31 @@ public TableDeclaration(GeneratorAttributeSyntaxContext context)
             throw new InvalidOperationException("Tagged enums cannot be tables.");
         }
 
-        var attrArgs = context.Attributes.Single().NamedArguments;
+        Visibility = context.TargetSymbol.DeclaredAccessibility switch {
+            Accessibility.ProtectedAndInternal
+            or Accessibility.NotApplicable
+            or Accessibility.Internal => "internal",
+            Accessibility.Public => "public",
+            _ => throw new Exception(
+                "Table row type visibility must be public or internal."
+            ),
+        };
 
-        IsPublic = attrArgs.Any(pair => pair is { Key: "Public", Value.Value: true });
+        Views = new(context.Attributes
+            .Where(a => a.AttributeClass?.ToDisplayString() == "SpacetimeDB.TableAttribute")
```

Comment:
You don't need this `.Where` - `context.Attributes` is already contextual attribute and is guaranteed to match what was passed in `ForAttributeWithMetadataName`.